lottie-swift/src/Private/LayerContainers/CompLayers/TextCompositionLayer.swift

```diff
@@ -91,8 +91,6 @@ final class TextCompositionLayer: CompositionLayer {
   override func displayContentsWithFrame(frame: CGFloat, forceUpdates: Bool) {
     guard let textDocument = textDocument else { return }
     
-    textLayer.contentsScale = self.renderScale
```

Comment:
I didn't test animations with Text Layers. Probably makes sense to check on that before merging

Reply:
Okay, so for text there seems to be a notable visible difference when removing contentsScale. I will have a look at how to set contentsScale for TextLayers 🤓